delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/indexing/CompositeIndexingStream.scala

```diff
@@ -372,15 +372,15 @@ object CompositeIndexingStream {
       indexingController.restart(id, project, Restart(PartialRestart(projections)))
     val remoteProjectCounts: RemoteProjectsCounts = source =>
       deltaClient
-        .projectCount(source)
+        .statistics(source)
```

Comment:
back to statistics? I thought I changed it in my previous PR to `projectCount`

Reply:
I thought that was before Simon made his change, I'll change it back